tests/integration_tests/performance/test_versioned_serialization_benchmark.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""A test that ensures that all unit tests pass at integration time."""
+
+import platform
+import os
+import logging
+import json
+
+import pytest
+import framework.utils as utils
+import host_tools.proc as proc
+from framework.defs import FC_WORKSPACE_DIR
+
+BENCHMARK_DIRECTORY = "{}/src/vmm".format(FC_WORKSPACE_DIR)
```

Comment:
Can we move the benchmark dir in `build/ ` so we do not dirty the source tree ?

Reply:
Criterion does not allow for a different Target directory path. Basically, running 'cargo bench' will create the results directory in the same location it was run from. I can move the folder manually if it is needed though.

Reply:
Apparently, it is possible. I will change the directory in this regard.